src/ui/public/modals/confirm_modal.html

```diff
@@ -1,4 +1,16 @@
 <div class="kuiModal" style="width: 450px" data-test-subj="confirmModal">
+  <div class="kuiModalHeader" ng-if="title">
```

Comment:
I think it looks weird without a title and an empty header.  It can't be that hard to come up with a title (`Warning`, `Error`, `Info` at the most basic), so I kinda like forcing people to add one, even if only for aesthetic reasons.  If you disagree I can make it optional though.